### PR TITLE
[docs] Update plugin docs for a few libraries

### PR DESCRIPTION
**Pull request by cruzach**
# Why and how

https://www.notion.so/expo/Config-plugin-and-permissions-docs-e0f5b8f5b98f4a6eb3ef279dcbd4de02

# Test Plan

run docs

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/admob.md [hunk 1]
```diff
@@ -14,7 +17,9 @@ Expo includes support for the [Google AdMob SDK](https://www.google.com/admob/)
 
 <InstallSection packageName="expo-ads-admob" />
 
-## Configuration
+## Configuration in app.json / app.config.js
+
+You can configure `expo-ads-admob` using its built-in [config plugin](../../../guides/config-plugins.md) if you use config plugins in your project ([EAS Build](../../../build/introduction.md) or `expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/admob.md:23]: i think we should also have a section here addressing people using classic builds, indicating how to configure the tracking permissions message

File: docs/pages/versions/unversioned/sdk/facebook-ads.md [hunk 2]
```diff
@@ -22,13 +25,49 @@ For bare apps, you will also need to follow [Facebook's Get Started guide](https
...
 
-### Configuring app.json
+### Configuration in app.json / app.config.js
+
+You can configure `expo-ads-facebook` using its built-in [config plugin](../../../guides/config-plugins.md) if you use config plugins in your project ([EAS Build](../../../build/introduction.md) or `expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect.
 
 In your project's [app.json](../../../workflow/configuration.md), add your [Facebook App ID and Facebook Display Name](https://developers.facebook.com/docs/facebook-login/ios) under the `facebookAppId` and `facebookDisplayName` keys.
 
 - In the Expo Go app, all of your Facebook API calls will be made with Expo's Facebook App ID. This means you will not see any related ad info in your Facebook developer page while running your project in Expo Go.
 - To use your app's own Facebook App ID (and thus see any related ad info in your Facebook developer page), you'll need to [build a standalone app](../../../distribution/building-standalone-apps.md).
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/facebook-ads.md:36]: same as above re: addressing classic builds for tracking permission message

File: docs/pages/versions/unversioned/sdk/media-library.md [hunk 3]
```diff
@@ -17,14 +19,67 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
...
 
-## Configuration
+## Configuration in app.json / app.config.js
 
-In managed apps, the permission to access images or videos is added automatically.
+You can configure `expo-media-library` using its built-in [config plugin](../../../guides/config-plugins.md) if you use config plugins in your project ([EAS Build](../../../build/introduction.md) or `expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect.
+
+<ConfigClassic>
+
+You can configure [the permissions for this library](#permissions) using [`ios.infoPlist`](../config/app.md#infoplist) and [`android.permissions`](../config/app.md#permissions).
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/media-library.md:28]: we can reuse this message for the tracking permissions stuff above i think

File: docs/pages/versions/unversioned/sdk/tracking-transparency.md [hunk 4]
```diff
@@ -18,6 +20,39 @@ For more information on Apple's new App Tracking Transparency framework, please
 
 <InstallSection packageName="expo-tracking-transparency" />
 
+## Configuration in app.json / app.config.js
+
+You can configure `expo-tracking-transparency` using its built-in [config plugin](../../../guides/config-plugins.md) if you use config plugins in your project ([EAS Build](../../../build/introduction.md) or `expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect.
+
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/tracking-transparency.md:26]: i'm a broken record at this point

**Review by brentvatne — APPROVED** [review]
looks good, just need to update to provide some information about customizing tracking permission message on classic builds

File: docs/pages/versions/unversioned/sdk/admob.md (hunk 1 above)
Comment (cruzach) [reply docs/pages/versions/unversioned/sdk/admob.md:23]: Sure, I can add it, but I thought the `IOSPermissions` block was for that?

Comment (cruzach) [reply docs/pages/versions/unversioned/sdk/admob.md:23]: added

**Pull request merged**